test/snippet/alphabet/quality/phred94.cpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/quality/phred94.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    seqan3::phred94 phred;
+    phred.assign_rank(2); // wrapper for assign_phred(2)
+    seqan3::debug_stream << phred.to_phred() << "\n"; // 2
+    seqan3::debug_stream << phred.to_char() << "\n";  // '#'
+    seqan3::debug_stream << phred.to_rank() << "\n";  // 2
+
+    seqan3::phred94 another_phred{75};
+    seqan3::debug_stream << another_phred.to_phred() << "\n"; // 75
+}
```

Comment:
What happens when you construct a phred of a higher value than 93 ? e.g. 105.
Can you check what happens?